python/ray/data/_internal/logging_json.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
+disable_existing_loggers: False
+
+formatters:
+    ray:
+        class: ray._private.ray_logging.formatters.JSONFormatter
+
+filters:
+    console_filter:
+        (): ray.data._internal.logging.HiddenRecordFilter
+    core_context_filter:
+        class: ray._private.ray_logging.filters.CoreContextFilter
+
+handlers:
+    file:
+        class: ray.data._internal.logging.SessionFileHandler
+        formatter: ray
+        filename: ray-data.log
+        filters: [core_context_filter]
+    console:
+        class: ray._private.log.PlainRayHandler
+        formatter: ray
+        level: INFO
+        filters: [console_filter, core_context_filter]
```

Comment:
Do we want to use the same behavior as TEXT for the console logger?

Reply:
Yes, ideally console should continue to be plain text while in the file is JSON.

Reply:
Updated code and examples above to log text to console regardless of mode.